restructure quota mgmt docs v2 (#5140)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

- **Transaction** - A [transaction](/product/performance/transaction-summary/#what-is-a-transaction) represents a single instance of a service being called to support an operation you want to measure or track, like a page load. Transaction events are grouped by the transaction name.
- **Quota** - Your quota is the monthly number of events — errors, attachments, and transactions — that you pay Sentry to track.

## Managing My Quota, an Overview

Sentry completes a thorough evaluation of each event to determine if it counts toward your quota, as outlined in this overview. Detailed documentation for each evaluation is linked throughout. Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed. In addition, for error events, Sentry validates that the event contains valid fingerprint information. If any of these items are missing or incorrect, the event is rejected.

### 1. Spike Protection

> Sentry’s spike protection prevents huge overages in error events from consuming your event capacity. Spike protection is not currently available for transactions. Learn more in [Spike Protection](#spike-protection).

Author: ```suggestion
Sentry’s spike protection prevents huge overages in error events from consuming your event capacity. Spike protection is not currently available for transactions. Learn more in the Spike Protection section of our [error quota management guide](/product/accounts/quotas/manage-event-stream-guide/#1-spike-protection). 
```

### 2. Event Usage Stats

> blah blah

Author: ```suggestion
blah blah

Learn more in the Event Usage Stats section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#common-workflows-for-managing-your-event-stream) and [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#1-event-usage-stats).
```

### 3. Quota Availability

> Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. Learn about adding to your quota and what happens when you exceed it in [Increasing Quotas](#increasing-quotas).

Author: ```suggestion
Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. Learn about adding to your quota and what happens when you exceed it in the Increasing Quotas section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#3-increasing-quotas) and [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#2-increasing-quotas). 
```

### 4. Rate Limits

> If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. Learn more in [Rate Limiting](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or in [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments. You can add limits for error events and attachments in [sentry.io](https://sentry.io). This does not apply to transaction events.

Author: ```suggestion
If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. You can add limits for error events and attachments in [sentry.io](https://sentry.io). This does not apply to transaction events. Learn more in the Rate Limiting section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#4-rate-limiting) or
```

#### Attachment Limits

If you have enabled the storage of crash reports, you may set limits for the maximum number of crash reports that will be stored per issue. To set up these limits, use the slider in the "Store Native Crash Reports" option in your organization's "Security & Privacy" settings.

> ### 6. Event Repetition

Author: ```suggestion
### 6. Delete & Discard Issues
```

Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:

> - If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.

Author: ```suggestion
- If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota. Learn more in the Delete & Discard section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#5-applying-workflows).
```

- If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.

> - If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).

Author: ```suggestion
- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in the Inbound Filters section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#6-inbound-data-filters). 
```

### 7. Inbound Filters

> If any inbound filter is set for a type of event or attachment, and your subscription allows, the event or attachment won't be counted. Learn more in [Inbound Data Filters](#inbound-data-filters).

Author: ```suggestion
If any inbound filter is set for a type of event or attachment, and your subscription allows, the event or attachment won't be counted. Learn more in the Inbound Data Filters section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#6-inbound-data-filters) and [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#3-inbound-data-filters).
```

After these checks are processed, the event counts toward your quota. It is accepted into Sentry, where it persists and is stored.

> ## 8. SDK Sample Rate

Author: ```suggestion
### 8. SDK Sample Rate
```

If a sample rate is defined for the SDK, the SDK evaluates whether this event should be sent as a representative fraction of events. Setting a sample rate is documented for each SDK. The SDK sample rate is not dynamic; changing it requires re-deployment. In addition, setting an SDK sample rate limits visibility into the source of events. Setting a rate limit for your project may better suit your needs.

### 9. SDK Filtering

> Blah blah

Author: ```suggestion
Blah blah

Learn More in the SDK Filtering section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#9-sdk-filtering-beforesend).
```

### 10. SDK Configuration

> The SDK configuration either allows the event or filters the event out. Learn more in [errors](/product/accounts/quotas/manage-event-stream-guide/) and [transactions](/product/accounts/quotas/manage-transaction-quota/) quota management docs or in the [filtering docs for your specific SDK](/platform-redirect/?next=/configuration/filtering/).

Author: ```suggestion
The SDK configuration either allows the event or filters the event out. Learn more in the SDK Configuration section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#10-sdk-configuration) and [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#2-sdk-configuration-tracing-options) or in the [filtering docs for your specific SDK](/platform-redirect/?next=/configuration/filtering/).
```
